jerry-ext/debugger/debugger-tcp.c

```diff
@@ -33,11 +32,19 @@ typedef SSIZE_T ssize_t;
 /* On Windows the invalid socket's value of INVALID_SOCKET */
 #define JERRYX_SOCKET_INVALID INVALID_SOCKET
 
-/* On Windows sockets have a SOCKET typedef */
-typedef SOCKET jerryx_socket;
-
+/*
+ * On Windows, socket functions has the following signature:
```

Comment:
> (minor) ... functions have ...

why? it's socket related functions

Reply:
Because that's how the English grammar works. Plural goes with "have". "Functions" is plural. The proper sentence should read as "On Windows, socket functions have the following signature:" (or "signatures")

Reply:
Thanks, I fixed it with `signatures`